projects/wipp-frontend-lib/src/public-api.ts

```diff
@@ -0,0 +1,124 @@
+
+// dynamic-content
+export * from './lib/dynamic-content/dynamic-content.component';
+export * from './lib/dynamic-content/unknown-dynamic.component';
+
+// not-found
+export * from './lib/not-found/not-found.component';
+
+// generic data
+export * from './lib/generic-data/generic-data-detail/generic-data-detail.component';
+export * from './lib/generic-data/generic-data-list/generic-data-list.component';
+
+// images-collection
+export * from './lib/images-collection/images-collection-detail/images-collection-detail.component';
+export * from './lib/images-collection/images-collection-list/images-collection-list.component';
+export * from './lib/images-collection/images-collection-new/images-collection-new.component';
+export * from './lib/images-collection/images-collection-template/images-collection-template.component';
+
+// job
+export * from './lib/job/job-detail/job-detail.component';
+
+// modal-error
+export * from './lib/modal-error/modal-error.component';
+
+// not-found
+export * from './lib/not-found/not-found.component';
+
+// notebook
+export * from './lib/notebook/notebook-detail/notebook-detail.component';
+export * from './lib/notebook/notebook-list/notebook-list.component';
+export * from './lib/notebook/notebook-template/notebook-template.component';
+
+// plugin 
+export * from './lib/plugin/plugin-detail/plugin-detail.component';
+export * from './lib/plugin/plugin-list/plugin-list.component';
+export * from './lib/plugin/plugin-new/plugin-new.component';
+
+// pyramid
+export * from './lib/pyramid/pyramid-detail/pyramid-detail.component';
+export * from './lib/pyramid/pyramid-list/pyramid-list.component';
+export * from './lib/pyramid/pyramid-template/pyramid-template.component';
+
+// pyramid-annotation
+export * from './lib/pyramid-annotation/pyramid-annotation-detail/pyramid-annotation-detail.component';
+export * from './lib/pyramid-annotation/pyramid-annotation-list/pyramid-annotation-list.component';
+export * from './lib/pyramid-annotation/pyramid-annotation-template/pyramid-annotation-template.component';
+
+// pyramid visualization
+export * from './lib/pyramid-visualization/pyramid-visualization-detail/pyramid-visualization-detail.component';
+export * from './lib/pyramid-visualization/pyramid-visualization-list/pyramid-visualization-list.component';
+
+// stitching-vector
+export * from './lib/stitching-vector/stitching-vector-detail/stitching-vector-detail.component';
+export * from './lib/stitching-vector/stitching-vector-list/stitching-vector-list.component';
+export * from './lib/stitching-vector/stitching-vector-new/stitching-vector-new.component';
+
+// tensorflow-model
+export * from './lib/tensorflow-model/tensorflow-model-detail/tensorflow-model-detail.component';
+export * from './lib/tensorflow-model/tensorflow-model-list/tensorflow-model-list.component';
+export * from './lib/tensorflow-model/tensorflow-model-template/tensorboard-logs-template.component';
```

Comment:
I am only exporting the `list` and `detail` components because the `template` and `new` components are accessible from the former mentioned components. So, in short, there is no need to export them. However, I think exporting them would be a good idea just in case these components are needed.